### ISSUE TITLE
Screenshot Improvements tweak not currently able to remove copyright text.

### ISSUE THREAD

**Issue by BtEtta** [Caraxi/SimpleTweaksPlugin#905]
The Remove copyright text feature of the Screenshot Improvements tweak is currently greyed out in the plugin options. Hovering over it gives the tooltip: "Failed to locate address needed for this option"

**Comment by Apohlo** [comment]
Yeah I'm getting this issue too. Just noticed the watermark in a screenshot I took, found the option in Simple Tweaks and it's greyed out for some reason?
